plugins/pastefromwordimage/plugin.js

```diff
@@ -8,9 +8,55 @@
 
 	CKEDITOR.plugins.add( 'pastefromwordimage', {
 		requires: 'pastefromword',
-		init: function() {}
+		init: function( editor ) {
+			if ( CKEDITOR.env.ie || CKEDITOR.env.iOS ) {
```

Comment:
Was there any particular reason why it was changed from `!CKEDITOR.plugins.clipboard.isCustomDataTypesSupported` to `CKEDITOR.env.ie || CKEDITOR.env.iOS`?

If it is about iOS you may always go with `!CKEDITOR.plugins.clipboard.isCustomDataTypesSupported || CKEDITOR.env.iOS` which is more flexible (as `CKEDITOR.plugins.clipboard.isCustomDataTypesSupported` may change at some point and then it will automatically work here too).

Reply:
I've change wrong file :( 
It supposed to be switched in manual test here: https://github.com/ckeditor/ckeditor-dev/blob/aa75e0874a372dd89168314eaae2162c248a1ff9/tests/plugins/pastefromwordimage/manual/pastewordwithimage.html#L3
To avoid problem with loading plugins. As manual test are usually fired only while releasing, then such change should be sufficient there.